L3/ViewControllers/MainViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  MainViewController.swift
+//  L3
+//
+//  Created by Andrii Kuznietsov on 04.12.2023.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
+    
+    private enum Const {
+        static let padding: CGFloat = 24
+    }
+    
+    private var cardTextField = CardTextField()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureViewContoller()
+        createDismissKeyboardTapGesture()
+        configureLayout()
+    }
+    
+    private func configureViewContoller() {
+        view.backgroundColor = .systemBackground
+        view.addSubview(cardTextField)
+    }
+    
+    private func configureLayout() {
+        cardTextField.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Let's make it more fancy, look, create extension for UIView for example .forAutolayout so you can write here 

```suggestion
        cardTextField.forAutolayout()
```